sdk/formrecognizer/ai-form-recognizer/src/documentAnalysisClient.ts

```diff
@@ -381,21 +393,21 @@ export class DocumentAnalysisClient {
     return this._tracing.withSpan(
       "DocumentAnalysisClient.beginAnalyzeDocumentFromUrl",
       options,
-      this.analyze.bind(this, model, documentUrl)
+      this.analyze.bind(this, model, source("url", documentUrl))
     );
   }
 
   /**
    * A helper method for running analysis polymorphically.
-   * @internal
```

Comment:
Do you remember why we had this tagged with @internal?

Reply:
Not sure, but there is a lint rule that tells me not to put `@internal` on private methods.